### PR TITLE
feat(core) Theme Config

### PR DESCRIPTION
**Pull request by siteshackinc**
Need to have SPEC file completed for tests.

### CODE REVIEW

File: modules/core/client/controllers/core.client.config.theme.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('core')
+    .config('theme');
```
Comment (trainerbill) [inline modules/core/client/controllers/core.client.config.theme.js]: This isn't gonna work.  theme is a function not a string

**Comment by trainerbill** [comment]
The file needs to go into the config folder, its not a controller.  I just pulled the branch and if you fix the issue with the theme above then it passes the tests.

**Comment by siteshackinc** [comment]
Will add the spec file later

**Comment by trainerbill** [comment]
@siteshackinc Why was this not rebased and squashed?  there are almost as many commits as lines of code.

**Pull request merged**